misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses_status/manufacturing.js

```diff
@@ -75,8 +75,10 @@ const computeLineStatusFromSteps = ({ draftLine }) => {
   const stepStatuses = [];
   stepIds.forEach((stepId) => {
     const draftStep = draftLine.steps[stepId];
-    if (!stepStatuses.includes(draftStep.completeStatus)) {
-      stepStatuses.push(draftStep.completeStatus);
+    const status = draftStep.qtyToIssue === 0 ? CompleteStatus.COMPLETED : draftStep.completeStatus;
```

Comment:
pls just add a comment about why we consider completed when draftStep.qtyToIssue === 0.
just to not forget why we did this and to kick out later.